### ISSUE TITLE
2025-12-20: sgl-project/sglang 仓库更新报告

### ISSUE THREAD

**Issue by yan97ao** [yan97ao/argus#530]
# 每日更新报告（2025-12-20）

## sgl-project/sglang

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-12-20 23:43:55 | Mick | [diffusion] log: fix wrong use of suppress_other_loggers (#15534) |
| 2025-12-20 23:14:55 | Mick | [diffusion] chore: allow all attention backends if not specified (#15530) |
| 2025-12-20 22:52:40 | Xiaoyu Zhang | [diffusion] perf: support zero-cost weight offload and overlap with compute for wan-series  (#15511) |
| 2025-12-20 22:49:06 | Yingchun Lai | fix MiMo-V2-Flash typo (#15536) |
| 2025-12-20 21:00:07 | Yuan Luo | [VLM] Support ViT Piecewise CUDA Graph for Qwen3-VL (#15320) |
| 2025-12-20 18:31:24 | mlmz | vlm: refactor engine vlm params and support processor output as input (#14091) |
| 2025-12-20 17:45:18 | Yuan Luo | Optimize MiMo-V2-Flash by flashinfer fused allreduce (#15464) |
| 2025-12-20 14:53:46 | Артем Савкин | [NPU] [BUGFIX] [CRITICAL!] Fix NPU inference (torch_npu._npu_reshape_and_cache() crash) (#15484) |
| 2025-12-20 14:19:09 | Hubert Lu | [AMD] Support fast_topk kernels in sgl-kernel (#15172) |
| 2025-12-20 14:10:46 | yctseng0211 | [AMD] remove the redundant projection (#15178) |
| 2025-12-20 12:32:13 | Qiaolin Yu | tiny fix sampling seed for completion api  (#15498) |
| 2025-12-20 12:30:44 | Yuxuan Zhang | [GLM-4.7] GLM-4.7 Tool Parser and Doc Update (#15333) |
| 2025-12-20 11:24:31 | Mick | [diffusion] refactor: deprecate workloadtype (#15267) |
| 2025-12-20 11:14:52 | Simo Lin | [model-gateway] Implement RAII load guard with response body attachment (#15507) |
| 2025-12-20 10:59:41 | Simo Lin | [ci] remove rust benchmark in unit test ci (#15510) |
| 2025-12-20 09:38:46 | Liangsheng Yin | Clean `hidden_states_before_norm` (#15485) |
| 2025-12-20 08:25:46 | kun-llfl | [router] bugfix: cache_aware in grpc inbalance forward (#15473) |
| 2025-12-20 08:23:02 | Simo Lin | [model-gateway] simplify workflow engine backoff and reduce duplicate reads (#15505) |
| 2025-12-20 07:49:51 | Simo Lin | [model-gateway] Run workflow event subscribers concurrently (#15504) |
| 2025-12-20 07:44:57 | Simo Lin | [model-gateway] Optimize workflow engine with pre-computed dependency graph (#15503) |
| 2025-12-20 05:35:28 | Simo Lin | [model-gateway] Improve logging across core modules (#15497) |
| 2025-12-20 05:23:33 | YL D | [DSv32] Move deep_gemm.get_paged_mqa_logits_metadata to init time as metadata (#15040) |
| 2025-12-20 05:15:19 | michael-amd | [AMD] Add TP=8 models to nightly test and make TP=2 test stable (#15296) |
| 2025-12-20 05:02:57 | Simo Lin | [model-gateway] Improve logging in policies module (#15496) |
| 2025-12-20 04:57:18 | Simo Lin | [model-gateway] Improve logging in data_connector module (#15495) |
| 2025-12-20 04:56:33 | Simo Lin | [model-gateway] refactor: extract common graceful shutdown code before TLS branch (#15494) |
| 2025-12-20 04:35:50 | Simo Lin | [model-gateway] fix graceful shutdown for TLS/Non-TLS server (#15491) |
| 2025-12-20 02:49:21 | Khush Gupta | fixed trtllm nvfp4 backend for moe (#15022) |
| 2025-12-20 01:59:27 | Yuwei An | EP Support for Piecewise Cuda Graph (#14164) |
| 2025-12-20 01:47:17 | Yashika Gandhi - Google | [diffusion] http-server: support vertex generate pathway (#15348) |
| 2025-12-20 00:25:57 | Liangsheng Yin | Enhance protection rules of code owners (#15406) |
